packages/headless/src/insight.index.ts

```diff
@@ -178,6 +178,7 @@ export type {
 } from './controllers/insight/generated-answer/headless-insight-generated-answer.js';
 export {buildGeneratedAnswer} from './controllers/insight/generated-answer/headless-insight-generated-answer.js';
 export type {
+  GeneratedAnswerCitation,
```

Comment:
If we want to match the already exported types,

This should instead be exported in `headless-insight-generated-answer.ts` and be exported in this index from there as well.